src/containers/StructurePage/resourceComponents/ResourceItems.tsx

```diff
@@ -0,0 +1,147 @@
+/**
+ * Copyright (c) 2017-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import React, { memo, useState } from 'react';
+import { injectT, tType } from '@ndla/i18n';
+import Resource from './Resource';
+import {
+  deleteTopicResource,
+  updateTopicResource,
+  updateTopicSubtopic,
+  updateSubjectTopic,
+} from '../../../modules/taxonomy';
+import handleError from '../../../util/handleError';
+import MakeDndList from '../../../components/MakeDndList';
+import AlertModal from '../../../components/AlertModal';
+import Spinner from '../../../components/Spinner';
+import { TopicResource } from './StructureResources';
+import { classes } from './ResourceGroup';
+
+interface Props {
+  resources: TopicResource[];
+  refreshResources: () => Promise<void>;
+  locale: string;
+}
+
+const ResourceItems = ({ refreshResources, resources, locale, t }: Props & tType) => {
+  const [deleteId, setDeleteId] = useState<string>();
+  const [error, setError] = useState<string>();
+  const [loading, setLoading] = useState<boolean>();
+
+  const onDelete = async (deleteId: string) => {
+    try {
+      setDeleteId('');
+      setError('');
+      await deleteTopicResource(deleteId);
+      refreshResources();
+    } catch (e) {
+      handleError(e);
+      setError(`${t('taxonomy.errorMessage')}: ${e.message}`);
+    }
+  };
+
+  const onDragEnd = async ({
+    destination,
+    source,
+  }: {
+    destination?: { index: number };
+    source: { index: number };
+  }) => {
+    if (!destination) {
+      return;
+    }
+    try {
+      const { connectionId, primary, relevanceId, rank: currentRank } = resources[source.index];
+      const { rank } = resources[destination.index];
+      if (currentRank === rank) {
+        return;
+      }
+
+      setLoading(true);
+      await updateTopicResource(connectionId, {
+        primary,
+        rank: currentRank > rank ? rank : rank + 1,
+        relevanceId,
+      });
+      await refreshResources();
+    } catch (e) {
+      handleError(e.message);
+    }
+    setLoading(false);
+  };
+
+  const toggleDelete = (newDeleteId: string) => {
+    setDeleteId(newDeleteId);
+  };
+
+  const updateRelevanceId = async (
+    connectionId: string,
+    body: {
+      primary?: boolean;
+      rank?: number;
+      relevanceId?: string;
+    },
+  ) => {
+    const [, connectionType] = connectionId.split(':');
```

Comment:
Gjør kommaet noe her? Såvidt jeg forstår kan man velge startindex på lista via komma, men her settes ikke en verdi. https://stackoverflow.com/questions/27792281/javascript-comma-after-opening-bracket-of-parameter-in-syntax-example

Reply:
`split(':')` returnerer et array med elementer. Ved å bruke 
```tsx
const [, connectionType] = connectionId.split(':');
```
denne syntaksen sier du at du omitter første elementet i arrayet og lager en variabel `connectionType` som skal inneholde den andre vardien fra arrayet. 

eks, la oss si at det som split returnerer ser slik ut `[1,2,3]`.
da får du 
```tsx
const [, connectionType] = [1,2,3]
```
både 1 og 3 blir omitta, og `connectionType` får verdien 2.

Kunne også brukt `connectionId.split(':')[1]` for samme resultat.

Reply:
Dette er syntax for å hente ut det andre elementet i en array!